packages/web-components/@sfx/base/src/base.ts

```diff
@@ -0,0 +1,72 @@
+import { LitElement, customElement, html } from 'lit-element';
+import { makeSlot, createChildrenObserver } from './utils';
+
+/**
+ * Extends LitElement and all components extend it.
```

Comment:
I would word it like so, which better communicates what its intention is, in my opinion:

```suggestion
 * A base component for all SF-X components to extend. It is based on LitElement.
```